cluster/src/main/java/com/hartwig/pipeline/calling/sage/SageCaller.java

```diff
@@ -26,6 +26,9 @@
 
 public abstract class SageCaller extends TertiaryStage<SageOutput> {
 
+    public static final String SAGE_GENE_COVERAGE_TSV = "sage.gene.coverage.tsv";
+    public static final String SAGE_BQR_PNG = "sage.bqr.png";
```

Comment:
Could these be `protected` or package-private?

Reply:
no they are used in test classes